### ISSUE TITLE
No error message is shown is a font is missing

### ISSUE THREAD

**Issue by kmatheussen** [kmatheussen/radium#1232]
Use QFontDatabse: https://doc.qt.io/qt-5/qfontdatabase.html

**Comment by kmatheussen** [comment]
@Teteros Radium does require Bitstream Vera Sans Mono and many other fonts. Arch linux should include them. The error message mentioned in #1115 is displayed by a 3rd party library and not by Radium. But Radium does use this font, and it should display a warning if the font is missing.

**Comment by kmatheussen** [comment]
(that error message from the 3rd party library about the missing vera sans mono font is displayed even if the font is installed)

**Comment by Teteros** [comment]
Good call, the comments on the AUR page were right then, indeed the error message from that lib is misleading.

> Radium does require Bitstream Vera Sans Mono and many other fonts.

I've added `ttf-bitstream-vera`, are there any other ones that come to mind?

**Comment by kmatheussen** [comment]
Yes, it should only use the fonts in bin/fonts. But there is no assertion that the fonts are actually available, so if it's not available, Qt will just silently find a different font (this needs to be fixed).

But when displaying native VST plugins, JUCE is used instead of Qt for the text buttons at the top (bypass-button etc.), and I haven't configured JUCE to look in bin/fonts for font files. I haven't checked, but it's possible that JUCE is using bitstream vera by default, so this font might have to be installed globally.
